tests/stream/test_stream_smoke/0098_fixed_issues2.yaml

```diff
@@ -340,3 +340,73 @@ tests:
       - query_id: fixed-issues2-1
         expected_results:
           - [2, '2024-05-16 21:32:00+00:00', 1]
+
+  - id: 4
+    tags:
+      - stateless_function
+    name: issue 835
+    description: display timezone information for datetime
+    steps:
+      - statements:
+          - client: python
+            query_type: table
+            query: drop stream if exists fixed_issues2_stream;
+
+          - client: python
+            query_type: table
+            exists: fixed_issues2_stream
+            exists_wait: 2
+            wait: 1
+            query: create stream fixed_issues2_stream(t1 datetime('Europe/London'), t2 datetime, t3 datetime('Australia/Sydney'));
+
+          - client: python
+            query_type: table
+            depends_on_stream: fixed_issues2_stream
+            wait: 2
+            query: insert into fixed_issues2_stream (t1, t2, t3) values (to_time('6/9/2024 13:28','Europe/London'), to_time('6/9/2024 13:28','Europe/London'), to_time('6/9/2024 13:28','Europe/London'));
+
+          - client: python
+            query_id: fixed-issues2-1
+            query_type: table
+            wait: 2
+            query: select t1, t2, t3 from table(fixed_issues2_stream);
```

Comment:
prefer to add functional test instead of smoke test in https://github.com/timeplus-io/proton/tree/c01bc3f18122361607b442dd4c7daa9fae3c5117/tests/queries_ported/0_stateless